projects/cds/src/profiletag/services/spartacus-event.service.spec.ts

```diff
@@ -18,6 +18,10 @@ import { tap } from 'rxjs/operators';
 import { CdsConfig } from '../../config';
 import { SpartacusEventService } from './spartacus-event.service';
 
+const TWO_TIMES_CALLED = 2;
+const FOUR_TIMES_CALLED = 4;
+const FIVE_TIMES_CALLED = 5;
```

Comment:
Instead of define const at the beginning of file, you can do it inside describe/it block

const EXPECTED_TIMES_CALLED = 2;
...
const EXPECTED_TIMES_CALLED = 4;
etc.